sustc-api/src/main/java/io/sustc/dto/UserRecord.java

```diff
@@ -1,44 +1,70 @@
 package io.sustc.dto;
 
-import com.opencsv.bean.CsvBindByName;
 import lombok.Data;
 
 @Data
 public class UserRecord {
 
-    @CsvBindByName(column = "Mid")
-    private Long mid;
+    /**
+     * The user's ID
+     */
+    private long mid;
 
-    @CsvBindByName(column = "Name")
+    /**
+     * The user's name
+     */
     private String name;
 
-    @CsvBindByName(column = "Sex")
+    /**
+     * The user's sex
+     */
     private String sex;
 
-    @CsvBindByName(column = "Birthday")
+    /**
+     * The user's birthday
+     */
     private String birthday;
 
-    @CsvBindByName(column = "Level")
-    private Short level;
+    /**
+     * The user's level
+     */
+    private int level;
 
-    @CsvBindByName(column = "Sign")
-    private String sign;
+    /**
+     * The user's current number of coins
+     */
+    private int coin;
```

Comment:
consider adding a comment on `VideoService#coinVideo` about how to earn a coin (by transferring a coin donated to the video owner?)

if the owner receives 0.1 coin per donation, make this field `float`

Reply:
To make things simpler, I suggest that we do not allow earning coins.